services/base-service/models/expert/api.js

```diff
@@ -13,6 +13,7 @@ const { openapi, json_only, user_can_edit, public_or_is_user } = require('../mid
 
 function subselect(req, res, next) {
   try {
+    // This top part is gibberish to me, is this really what we want?
```

Comment:
@qjhart 
If I remember correctly, this was added to support both query and url params in get/post requests, but from the client perspective, we only ever make post requests.